draccus/wrappers/docstring.py

```diff
@@ -18,6 +18,28 @@ class AttributeDocString:
     docstring_below: str = ""
 
 
+# Default Help Orderings
+HELP_ORDERS = {
+    "inline": ["comment_inline", "comment_above", "docstring_below"],
+    "above": ["comment_above", "comment_inline", "docstring_below"],
+    "below": ["docstring_below", "comment_inline", "comment_above"],
+}
+
+
+def get_preferred_help_text(doc: AttributeDocString, preferred_help: str = "inline") -> Optional[str]:
+    if doc is None:
```

Comment:
should it be Optional[AttributeDocString] then?

Reply:
Oh good catch; by the semantics of `get_attribute_docstring` you'll never get a null AttributeDocString, it'll just be empty. Updating.